tools/rna_tools/intarna/intarna.xml

```diff
@@ -374,6 +471,32 @@
             </conditional>
             <output name="outfile" file="intarna_result.tabular"/>
         </test>
+        <test>
```

Comment:
@martin-raden we need to add a bunch of tests, so this gets automatically tested.

As you will see now this crashes in 1/2 tests. So the basic mode is working but the advanced mode is broken.

We use planemo (http://planemo.readthedocs.io/) to test such tools. `planemo test intarna.xml`